addons/crm/tests/common.py

```diff
@@ -616,6 +631,11 @@ def setUpClass(cls):
             'partner_id': cls.contact_company_1.id,
             'contact_name': 'Hermes Conrad',
             'email_from': 'hermes.conrad@test.example.com',
+            'tag_ids': [
+                (4, cls.tags[0].id),
+                (4, cls.tags[1].id),
+                (0, 0, {'name': 'Tag 5'}),
```

Comment:
Do we need that much data ? In any way, would create all tags first, then give them to sub records (0-3 to lead_1, 0-1-4 to lead_w_partner_company, ...). That way we can always compare records based on created data.